ui/apps/platform/src/Containers/NetworkGraph/NetworkGraph.tsx

```diff
@@ -21,7 +21,8 @@ import defaultComponentFactory from './components/defaultComponentFactory';
 import DeploymentSideBar from './deployment/DeploymentSideBar';
 import NamespaceSideBar from './namespace/NamespaceSideBar';
 import CidrBlockSideBar from './cidr/CidrBlockSideBar';
-import ExternalEntitiesSideBar from './external/ExternalEntitiesSideBar';
+import ExternalEntitiesSideBar from './externalEntities/ExternalEntitiesSideBar';
+import ExternalToClusterSideBar from './external/ExternalToClusterSideBar';
```

Comment:
These two names
```
ExternalEntitiesSideBar
ExternalToClusterSideBar
```

are a little too similar to parse easily.
What are some other names we could use?

Reply:
Honestly, External Entities always seemed like an odd name for a single node because it's plural. If I had to think of another name, maybe we could change `ExternalToCluster` to `ExternalGroup`?